core-web/apps/dotcms-ui/src/app/app-routing.module.ts

```diff
@@ -70,6 +70,18 @@ const PORTLETS_ANGULAR: Route[] = [
         loadChildren: () =>
             import('@dotcms/portlets/dot-locales/portlet').then((m) => m.DotLocalesRoutes)
     },
+    {
+        path: 'content-analytics',
+        canActivate: [MenuGuardService],
+        canActivateChild: [MenuGuardService],
+        data: {
+            reuseRoute: false
+        },
+        loadChildren: () =>
```

Comment:
maybe later change it to `loadComponent`, this route never will have child routes.

Reply:
done